src/module/skill-actions.ts

```diff
@@ -32,6 +33,7 @@ export default class SkillAction {
     const skill = actor.data.data.skills[proficiencyKey];
 
     this.key = key;
+    this.itemName = label;
     this.actor = actor;
     this.label = game.i18n.localize(skill.label) + ': ' + label;
```

Comment:
I think at some point it would be nice to make `this.label` calculated property since it depends on `skill` and `label`.